extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/GrpcServerRecorder.java

```diff
@@ -173,6 +174,8 @@ private void buildGrpcServer(Vertx vertx, GrpcServerConfiguration configuration,
             }
         });
         shutdown.addShutdownTask(route::remove); // remove this route at shutdown, this should reset it
+
+        initHealthStorage();
```

Comment:
@alesj This fixed the class loading problem - we need to initialize the `GrpcHealthStorage` eagerly for `quarkus.grpc.server.use-separate-server=false` (if needed).

Reply:
This is strange ... but ... if it's needed, then sure.

Reply:
Yes, it is. But class loading in the dev mode (+ dev ui) is tricky.